### PR TITLE
#159987629 User should be able to create articles

### PR DESCRIPTION
**Pull request by emmaadesile**
#### What does this PR do?
Users should be able to create articles

#### Description of Task to be completed?
- users should be able to create articles
- users should be able to add images to their articles
- users should be able to format their text
- users should be able to publish their

#### How should this be manually tested?
- run `git clone <branch url>
- npm install to install app dependencies
- run npm run dev to start the development server
- login to the app
- navigate to `localhost:3000/create-article`
- after writing the article, click publish

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[159987629](https://www.pivotaltracker.com/story/show/159987643)

### CODE REVIEW

File: src/containers/CreateArticle/TextEditor.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import React, { Component } from "react";
+import { Editor } from "@tinymce/tinymce-react";
+import PropTypes from "prop-types";
+
+const apiKey = "0naso5vy7fw5eu2eqblor00o1c326x0hupermg4cli0rqoqn";
```
Comment (d-beloved) [inline src/containers/CreateArticle/TextEditor.js]: please move this into the environment variable

**Review by d-beloved — CHANGES_REQUESTED** [review]
A very good job you have done so far.

Please fix the comment I made.

File: src/containers/CreateArticle/TextEditor.js (hunk 1 above)
Comment (emmaadesile) [reply src/containers/CreateArticle/TextEditor.js]: Done

**Review by emekafredy — APPROVED** [review]
LGTM

**Review by d-beloved — APPROVED** [review]

**Review by johnkingzy — APPROVED** [review]
Nice one!

**Pull request merged**